### ISSUE TITLE
Nested lists remove bottom margin

### ISSUE THREAD

**Issue by adammichaelwood** [opendatakit/docs#538]
Lists have a margin on the bottom. But a list inside a list does not. This almost always looks bad.

![screen shot 2018-02-27 at 8 39 24 am](https://user-images.githubusercontent.com/1776241/36741406-e113b016-1b99-11e8-99d7-1c24ac76714f.png)

Before working on this, I recommend looking through the docs and finding many examples of nested lists --- simple nested lists, lists inside paragraphs inside lists, etc. It may be worthwhile to create an ad-hoc list examples page for testing. We want nested lists to have sensible and consistent spacing. We do not want some of them to suddenly have double (or more) spacing.

**Comment by adammichaelwood** [comment]
Here's another example of nested lists looking terrible because of spacing.

![screen shot 2018-02-27 at 11 18 43 am](https://user-images.githubusercontent.com/1776241/36749592-13e15b22-1bb0-11e8-8790-ea5924796b92.png)

**Comment by ankita240796** [comment]
I found the same issue while adding show/hide feature for style-guide code-blocks. So, I added a line break along with the label to fix that. I am trying to find the reason for this.

**Comment by adammichaelwood** [comment]
Yeah, I've added line breaks here and there but we need a better solution in the CSS.

The *reason* for this problem, ultimately, is that the creators of the template didn't really envision list items containing paragraphs. (This oversight is pretty common.)

**Comment by ankita240796** [comment]
So, we need to play with the CSS to fix that. I will try doing that!

**Comment by adammichaelwood** [comment]
Essentially yes.

The CSS first defines lists as having some margin, and then later defines nested lists as not having it. I *think* the solution is to make nested lists match top-level lists. (I don't know if that will cause a range of other problems, so it might require fine tuning --- like only adding it if the containing list has paragraphs or something.

(Side note.... this makes me start to wonder: At what point should we think about making these changes in the SASS and compiling CSS as part of the build? And at what point should we start maintaining our own fork of rtd_theme? I'm not sure we're at that point yet, but we might get there before we know it. There are *a lot* of typography issues I'd like to deal with that are too hard to handle in an after-the-fact CSS monkey-patch approach)

**Comment by ankita240796** [comment]
I will look into all of this and share any queries here!

**Comment by adammichaelwood** [comment]
Another example, notable because an admonition block looks even worse after a nested a list:

![Nested list followed by an admonition block](https://user-images.githubusercontent.com/32369/37171801-e28ce92c-22c3-11e8-9fdf-53520e550cf1.png)
